samples/zephyr-hello/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+BOARD_ARCH=arm_cortex_m3
+BOARD_NAME=qemu_cortex_m3
+export ZEPHYR_BASE="PLEASE_SET_ZEPHYR_BASE"
```

Comment:
Better read ZEPHYR_BASE from environment, and only do this if no default provided.